src/main/java/de/gesundkrank/fzf4j/models/Result.java

```diff
@@ -77,4 +77,16 @@ public boolean isMatch() {
     public int getItemIndex() {
         return itemIndex;
     }
+
+    @Override
+    public String toString() {
+        return "Result{"
+               + "text='" + text + '\''
+               + ", start=" + start
+               + ", end=" + end
+               + ", score=" + score
+               + ", positions=" + Arrays.toString(positions)
+               + ", itemIndex=" + itemIndex
+               + '}';
+    }
```

Comment:
I know you don't like it, but lombok is great.

Reply:
Yes. I don't like it. 😏